zrml/prediction-markets/src/tests.rs

```diff
@@ -2909,99 +2909,6 @@ fn dispute_fails_unless_reported_or_disputed_market(status: MarketStatus) {
     });
 }
 
-#[test]
-fn it_resolves_a_disputed_market_to_default_if_dispute_mechanism_failed() {
```

Comment:
Could you kindly explain why you removed this test?

Reply:
main branch does not have these tests. https://github.com/zeitgeistpm/zeitgeist/commit/e991d9edbc76db6f2ecc2780a7caf4a4702cff6a#diff-ad4c4ea3d698c97bb435c01c3c30e79c5a60b34af1448fe64b7c87017054b4c6L2248

Reply:
I see, I even think that I deleted them.. But it had the reason that those tests were duplicated. Thanks for the investigation.